.github/workflows/build-dev-release.yml

```diff
@@ -1,17 +1,14 @@
-name: Build and publish dev-release Java distributions to sonatype.
+name: Build and publish dev release jar to sonatype repository
 
-on:
-  push:
-    tags:
-      - \d+.\d+.\d+-dev
+on: workflow_dispatch
```

Comment:
You won't be able to execute this workflow until it is merged into the default branch. So, we either should keep trigger on tag push or make the `dev-version-5` branch a default branch